### ISSUE TITLE
Error displaying points. Chart returns always the MAX value for the day

### ISSUE THREAD

**Issue by fagottino** [aeyrium/bezier-chart#74]
Hi @diegoveloper, i'm trying to display points on chart every 15 minutes.

I'm using the bezierChartAggregation AVERAGE with this DataSet:

```
data: [
DataPoint<DateTime>(value: 29, xAxis: DateTime(2020, 03, 17, 16, 29)),
DataPoint<DateTime>(value: 25, xAxis: DateTime(2020, 03, 18, 22, 22)),
DataPoint<DateTime>(value: 19.5, xAxis: DateTime(2020, 03, 20, 13, 38)),
DataPoint<DateTime>(value: 17.65, xAxis: DateTime(2020, 03, 21, 13, 38)),
DataPoint<DateTime>(value: 14.32, xAxis: DateTime(2020, 03, 22, 09, 38)),
DataPoint<DateTime>(value: 34.5, xAxis: DateTime(2020, 03, 22, 16, 46)),
DataPoint<DateTime>(value: 45.5,xAxis: DateTime(2020, 03, 23, 03, 46)),
DataPoint<DateTime>(value: 28.5, xAxis: DateTime(2020, 03, 23, 05, 05)),
DataPoint<DateTime>(value: 26.5, xAxis: DateTime(2020, 03, 23, 10, 23)),
DataPoint<DateTime>(value: 18.5, xAxis: DateTime(2020, 03, 23, 09, 17)),
]
```

**The current behaviour** is this:
For the day 03/22 i have 2 values: 14.32 and 34.5 but on the graph for the same day,  all day long, i have only the first between the two (as you can see in the screenshot) even if i don't have the values for other ranges.

**The behaviour i'm expecting** should be:
All 0 values for the missing timing in the DataSet and only 2 values for the point that DataSet contain (14.32 at about 09:38am and 34.5 at about 4:46pm) 

I hope that the screenshot in attachment will be useful.

Thank you for the support.

<img width="399" alt="Schermata 2020-03-23 alle 11 26 08" src="https://user-images.githubusercontent.com/5007007/77307368-67a8e580-6cf9-11ea-9afe-502e755e005d.png">
<img width="400" alt="Schermata 2020-03-23 alle 11 26 19" src="https://user-images.githubusercontent.com/5007007/77307380-6ecff380-6cf9-11ea-9284-5449a05d5a1d.png">
<img width="397" alt="Schermata 2020-03-23 alle 11 26 42" src="https://user-images.githubusercontent.com/5007007/77307397-7394a780-6cf9-11ea-8952-22180d224bbe.png">
<img width="398" alt="Schermata 2020-03-23 alle 11 26 50" src="https://user-images.githubusercontent.com/5007007/77307405-77282e80-6cf9-11ea-9570-5cce71e31233.png">

This is the modified library

[bezier_chart_widget.txt](https://github.com/aeyrium/bezier-chart/files/4368652/bezier_chart_widget.txt)

**Comment by diegoveloper** [comment]
Again , try to reproduce the issue in a simple code that I could run on my
own and paste here.

**Comment by fagottino** [comment]
Hi @diegoveloper, here is the files ready for you to try.
Just change the extension from txt to dart.

Thanks!

[bezier_chart_config.txt](https://github.com/aeyrium/bezier-chart/files/4370502/bezier_chart_config.txt)
[bezier_chart_widget.txt](https://github.com/aeyrium/bezier-chart/files/4370503/bezier_chart_widget.txt)
[main.txt](https://github.com/aeyrium/bezier-chart/files/4370504/main.txt)

**Comment by diegoveloper** [comment]
Ah sorry, I noticed you modified the package, I can't help you now.  PR are
open.

**Comment by diegoveloper** [comment]
@fagottino Please update the title of this issue

**Comment by diegoveloper** [comment]
@fagottino I'm confused because you mentioned on the title 'Error displaying points ...'  is this actually an error of the package?  because I see you modified the code , so it's not clear for me.

Is this an issue or you want a new feature for this package?
Did you try using `BezierChartScale.HOURLY` ?

**Comment by xxz3** [comment]
No, is not an error of the package. As me and @fagottino are trying to say, we are adding a new scale to the package, QUARTER, because HOURLY is not enough. BUT there is a problem showing the actual value for every point.

**Comment by fagottino** [comment]
@diegoveloper your library is working fine, but we want to add QUARTER to SCALE to show data every 15 minutes. So the error come from this situation.
Could you help us to improve your library?

**Comment by diegoveloper** [comment]
@fagottino you can change the title of this issue then, like 'Feature Request: Add a new scale .... '   . 
I'm working now but I'll check when I have time . Thanks
